nvm.sh

```diff
@@ -1021,23 +1021,23 @@ nvm_ls() {
       PATTERN='v'
       SEARCH_PATTERN='.*'
     else
-      SEARCH_PATTERN="$(echo "${PATTERN}" | command sed "s#\.#\\\.#g;")"
+      SEARCH_PATTERN="$(echo "${PATTERN}" | command sed "s#\\.#\\\\.#g;")"
     fi
     if [ -n "${NVM_DIRS_TO_SEARCH1}${NVM_DIRS_TO_SEARCH2}${NVM_DIRS_TO_SEARCH3}" ]; then
       VERSIONS="$(command find "${NVM_DIRS_TO_SEARCH1}"/* "${NVM_DIRS_TO_SEARCH2}"/* "${NVM_DIRS_TO_SEARCH3}"/* -name . -o -type d -prune -o -path "${PATTERN}*" \
         | command sed -e "
             s#${NVM_VERSION_DIR_IOJS}/#versions/${NVM_IOJS_PREFIX}/#;
             s#^${NVM_DIR}/##;
-            \#^[^v]# d;
-            \#^versions\$# d;
+            \\#^[^v]# d;
+            \\#^versions\$# d;
             s#^versions/##;
             s#^v#${NVM_NODE_PREFIX}/v#;
-            \#${SEARCH_PATTERN}# !d;
+            \\#${SEARCH_PATTERN}# !d;
           " \
-          -e "s#^\([^/]\{1,\}\)/\(.*\)\$#\2.\1#;" \
+          -e "s#^\\([^/]\\{1,\\}\\)/\\(.*\\)\$#\\2.\\1#;" \
         | command sort -t. -u -k 1.2,1n -k 2,2n -k 3,3n \
         | command sed "
-            s#\(.*\)\.\([^\.]\{1,\}\)\$#\2-\1#;
+            s#\\(.*\\)\\.\\([^\\.]\\{1,\\}\\)\$#\\2-\\1#;
```

Comment:
same here

Reply:
This one combined with shell variable below, I'm not very sure how to deal with it lol